modules/planning/tasks/deciders/path_decider_obstacle_utils.cc

```diff
@@ -0,0 +1,50 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#include "modules/planning/tasks/deciders/path_decider_obstacle_utils.h"
+
+namespace apollo {
+namespace planning {
+
+constexpr double kStaticObstacleSpeedThreshold = 0.5;
```

Comment:
This becomes a global variable.
Use `static` or anonymous namespace to make it local.

Reply:
Actually, this is meant to be global. Later I will put all constexpr into GFLAGS.